packages/config/config/devices/0x000c/hs-fls100-g2.json

```
@@ -0,0 +1,150 @@
+// HomeSeer Technologies HS-FLS100-G2
+// Floodlight Sensor
+{
+	"manufacturer": "HomeSeer Technologies",
+	"manufacturerId": "0x000c",
+	"label": "HS-FLS100-G2",
+	"description": "Floodlight Sensor",
+	"devices": [
+		{
+			"productType": "0x0201",
+			"productId": "0x000c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "PIR trigger off period",
+			"description": "Period to send Trigger Off command after PIR is triggered.",
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 8,
+			"maxValue": 720,
+			"defaultValue": 180,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Lux sensor threshold",
+			"description": "Lux level to activate the PIR",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 900,
+			"defaultValue": 50,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Auto Report luminance and temperature interval",
+			"description": "set the interval of periodic lux level report and temperature report to the controller",
+			"unit": "min",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 1440,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "PIR trigger alert",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "disable alert",
+					"value": 0
+				},
+				{
+					"label": "enable alert",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Floodlight control mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "controlled by Z wave controller only",
+					"value": 0
+				},
+				{
+					"label": "controlled by PIR trigger and Lux level, or by Z wave controller",
+					"value": 1
+				}
+			]
+		},
+		"6": {
+			"label": "Lux sensor",
```

```suggestion
			"label": "Floodlight triggers",
```
maybe? Lux sensor does not make sense in this context

---

Definitely agree that makes more sense.